pack/tdc/tdcp.json

```diff
@@ -70,5 +522,1050 @@
+        "cost": 1,
+        "faction_code": "neutral",
+        "illustrator": "Aleksander Karcz",
+        "myriad": true,
+        "name": "Sacred Oath",
+        "pack_code": "tdcp",
+        "position": 121,
+        "quantity": 1,
+        "restrictions": "trait:cultist, trait:cursed, trait:silver twilight",
+        "skill_agility": 1,
+        "skill_wild": 1,
+        "skill_willpower": 1,
+        "slot": "Arcane",
+        "subname": "Oath of Loyalty",
+        "text": "[[Cultist]], [[Cursed]], [[Silver Twilight]] deck only. Fast. Myriad. Limit 1 Sacred Oath <i>(Oath of Loyalty)</i> per deck.\nMax 3 copies of Sacred Oath in play. Up to 3 Sacred Oath cards take up 1 arcane slot.\n[reaction] When you draw a non-weakness [[Pact]] or [[Terror]] treachery, discard a copy of Sacred Oath from play: Cancel that treachery's revelation effect and discard it.",
+        "traits": "Pact.",
+        "type_code": "asset",
+        "xp": 5
+    },
+    {
+        "code": "11122",
+        "deck_limit": 1,
+        "cost": 1,
+        "faction_code": "neutral",
+        "illustrator": "Aleksander Karcz",
+        "myriad": true,
+        "name": "Sacred Oath",
+        "pack_code": "tdcp",
+        "position": 122,
+        "quantity": 1,
+        "restrictions": "trait:cultist, trait:cursed, trait:silver twilight",
+        "skill_combat": 1,
+        "skill_wild": 1,
+        "skill_willpower": 1,
+        "slot": "Arcane",
+        "subname": "Oath of Order",
+        "text": "[[Cultist]], [[Cursed]], [[Silver Twilight]] deck only. Fast. Myriad. Limit 1 Sacred Oath <i>(Oath of Order)</i> per deck.\nMax 3 copies of Sacred Oath in play. Up to 3 Sacred Oath cards take up 1 arcane slot.\n[reaction] When you draw a non-weakness [[Curse]] or [[Hex]] treachery, discard a copy of Sacred Oath from play: Cancel that treachery's revelation effect and discard it.",
+        "traits": "Pact.",
+        "type_code": "asset",
+        "xp": 5
+    },
+    {
+        "code": "11123",
+        "deck_limit": 1,
+        "cost": 1,
+        "faction_code": "neutral",
+        "illustrator": "Aleksander Karcz",
+        "myriad": true,
+        "name": "Sacred Oath",
+        "pack_code": "tdcp",
+        "position": 123,
+        "quantity": 1,
+        "restrictions": "trait:cultist, trait:cursed, trait:silver twilight",
+        "skill_intellect": 1,
+        "skill_wild": 1,
+        "skill_willpower": 1,
+        "slot": "Arcane",
+        "subname": "Oath of Wisdom",
+        "text": "[[Cultist]], [[Cursed]], [[Silver Twilight]] deck only. Fast. Myriad. Limit 1 Sacred Oath <i>(Oath of Wisdom)</i> per deck.\nMax 3 copies of Sacred Oath in play. Up to 3 Sacred Oath cards take up 1 arcane slot.\n[reaction] When you draw a non-weakness [[Omen]] or [[Power]] treachery, discard a copy of Sacred Oath from play: Cancel that treachery's revelation effect and discard it.",
+        "traits": "Pact.",
+        "type_code": "asset",
+        "xp": 5
```

Comment:
Are the Sacred Oaths going to mess up the db? You spend 5xp for 3 copies, but each copy is limit 1 per deck - so the db will want you to spend 15xp for all three... right? What's the work-around though? Changing them all to limit 3 per deck and having players pretend they have chosen one of each oath? I imagine these cards are also quantity 1 each...

Reply:
@drawntotheflame I have no idea tbh, it depends on the implementation - if DB implements the Myriad rule as written, it should work. That said, I had to fix it in my own implementation though.